src/main/webapp/app/shared/metis/posting-reactions-bar/post-reactions-bar/post-reactions-bar.component.html

```diff
@@ -65,6 +43,28 @@
             [darkMode]="darkModeEmojiPicker"
         ></emoji-mart>
     </ng-template>
+    <div class="col-auto p-0" *ngIf="displayPriority === DisplayPriority.PINNED || currentUserIsAtLeastTutor">
+        <button
+            class="reaction-button pin"
+            [class.reaction-button--not-hoverable]="!currentUserIsAtLeastTutor"
+            [class.reaction-button--reacted]="displayPriority === DisplayPriority.PINNED && currentUserIsAtLeastTutor"
+            [disabled]="!currentUserIsAtLeastTutor"
+            (click)="currentUserIsAtLeastTutor && togglePin()"
```

Comment:
Is this required? 🤔 
Seems like the button is disabled already, is there a reason for this check?
```suggestion
            (click)="togglePin()"
```